### PR TITLE
[chore] Improve SOPs

### PR DESCRIPTION
**Pull request by lgarciaaco**
- Create single SOP for Target Down or Pod Restarting alert. For Pod Restarting, point to Target Down since the procedure is the same.
- Add Escalate action at the end of every SOP
- Add alternative way to find failing connectors using plain `oc` commands.
- Mention that admin access to backplain might be required in case a Deployment needs to be forced rolled.
- Match AlertName and SopName, best effort.

A follow up PR is required deleting old SOPs and pointing alerts to the new SOPs

### CODE REVIEW

File: observability/sops/alerts/connectors_availability_error_budget_burnt.adoc [hunk 1]
```diff
@@ -214,6 +214,8 @@ oc get Pod -l cos.bf2.org/connector.id=<connectorId> -n redhat-openshift-connect
 oc logs -l cos.bf2.org/connector.id=<connectorId> -n redhat-openshift-connectors-<namespaceId> -f
 ----
 
-== Further Reading
+== Escalate to engineering
 
-Put any extra information about the alert here. I.e. Information that isn't critical to resolving to alert but might help SRE who's not familiar with the alert understand it and it's implications if they want to.
+* If the above hasn't worked contact engineering.
+** Use the ROHC  Run The Service https://coreos.slack.com/archives/C03AC361WCA[channel in slack].
```
Comment (pat-cremin) [inline observability/sops/alerts/connectors_availability_error_budget_burnt.adoc:220]: Change to:

Use the RHOC  Run The Service channel in slack: #rhoc-rts

File: observability/sops/alerts/container_down.adoc [hunk 2]
```diff
@@ -82,4 +82,6 @@ N/A
 
 == Escalate to engineering
 
-* If the above hasn't worked contact engineering.
\ No newline at end of file
+* If the above hasn't worked contact engineering.
+** Use the ROHC  Run The Service https://coreos.slack.com/archives/C03AC361WCA[channel in slack].
```
Comment (pat-cremin) [inline observability/sops/alerts/container_down.adoc:86]: Change to:

Use the RHOC Run The Service channel in slack: #rhoc-rts

File: observability/sops/alerts/cos_fleetshard_operator_camel_target_down.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+oc logs <pod_name> -n <namespace>
+----
+
+== Troubleshooting
+N/A
+
+== Escalate to engineering
+
+* If the above hasn't worked contact engineering.
+** Use the ROHC  Run The Service https://coreos.slack.com/archives/C03AC361WCA[channel in slack].
```
Comment (pat-cremin) [inline observability/sops/alerts/cos_fleetshard_operator_camel_target_down.adoc:86]: Change to:

Use the RHOC Run The Service channel in slack: #rhoc-rts

File: observability/sops/alerts/cos_fleetshard_operator_debezium_target_down.adoc [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+oc logs <pod_name> -n <namespace>
+----
+
+== Troubleshooting
+N/A
+
+== Escalate to engineering
+
+* If the above hasn't worked contact engineering.
+** Use the ROHC  Run The Service https://coreos.slack.com/archives/C03AC361WCA[channel in slack].
```
Comment (pat-cremin) [inline observability/sops/alerts/cos_fleetshard_operator_debezium_target_down.adoc:86]: Change to:

Use the RHOC Run The Service channel in slack: #rhoc-rts

File: observability/sops/alerts/cos_fleetshard_sync_target_down.adoc [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+oc logs <pod_name> -n <namespace>
+----
+
+== Troubleshooting
+N/A
+
+== Escalate to engineering
+
+* If the above hasn't worked contact engineering.
+** Use the ROHC  Run The Service https://coreos.slack.com/archives/C03AC361WCA[channel in slack].
```
Comment (pat-cremin) [inline observability/sops/alerts/cos_fleetshard_sync_target_down.adoc:86]: Change to:

Use the RHOC Run The Service channel in slack: #rhoc-rts

File: observability/sops/alerts/rhoc_connectors_availability.adoc [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+oc get Deployment -l cos.bf2.org/connector.id=<connectorId> -n redhat-openshift-connectors-<namespaceId>
+oc get ReplicaSet -l cos.bf2.org/connector.id=<connectorId> -n redhat-openshift-connectors-<namespaceId>
+oc get Pod -l cos.bf2.org/connector.id=<connectorId> -n redhat-openshift-connectors-<namespaceId>
+oc logs -l cos.bf2.org/connector.id=<connectorId> -n redhat-openshift-connectors-<namespaceId> -f
+----
+
+== Escalate to engineering
+
+* If the above hasn't worked contact engineering.
+** Use the ROHC  Run The Service https://coreos.slack.com/archives/C03AC361WCA[channel in slack].
```
Comment (pat-cremin) [inline observability/sops/alerts/rhoc_connectors_availability.adoc:236]: Change to:

Use the RHOC Run The Service channel in slack: #rhoc-rts

File: observability/sops/alerts/rhoc_connectors_availability.adoc [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+[NOTE]
+`cos_fleetshard_sync_connector_state_count_total` is reported by the cos-fleetshard-sync. Problems with the cos-fleetshard-sync could
+greatly affect the accuracy of this metric. Restarting the cos-fleetshard-sync pod will reset the `cos_fleetshard_sync_connector_state_count_total`
+counter and possibly tamper with the rate if the period of time used to calculate includes the restart.
+
+More info on this concept of multi-window multi-burn-rate alerts can be found
+https://sre.google/workbook/alerting-on-slos/#6-multiwindow-multi-burn-rate-alerts[here].
+
+More info on the SLO related to these alerts can be found
+https://docs.google.com/document/d/1GS8SROrDSZqWy6DD03TjJd0BM6OyLipv_BxAVb32XYk/edit#[here].
```
Comment (pat-cremin) [inline observability/sops/alerts/rhoc_connectors_availability.adoc:52]: Is this the right document for the connectors or does it share the doc with Kafka?

Comment (lgarciaaco) [reply observability/sops/alerts/rhoc_connectors_availability.adoc]: Copy/Paste left over

**Review by pat-cremin — APPROVED** [review]
LGTM

**Pull request merged**